install.sh

```diff
@@ -3349,6 +3391,10 @@ install_hooks_into_repo() {
 #   None
 ############################################################
 setup_shared_hook_repositories() {
+    if [ "$USE_HOOKS_PATH" = "yes" ]; then
```

Comment:
While the template files might come from the central folder, shared repos are a different story and can be useful here - maybe I need to explain these better in the readme.
Regardless of where the hooks scripts run from (templates or `core.hooksPath`), they will look for things to execute in `.githooks/<event>/*` within the repository, __plus__ there can be other hooks coming from shared hook repos which should run for all repositories. Does this make more sense?

Reply:
Please drop this new block.